### PR TITLE
[CFD-47] Admin Absence Page

### PR DESCRIPTION
**Pull request by helenaglow**
Made a page so that an admin can see all the absences on a given day.

Other changes:
-  Imported [react-datepicker](https://www.npmjs.com/package/react-datepicker). It could also be helpful in the future

### Reason for Change
Completes CFD-47

### Type of change
- [ X] New feature (non-breaking change which adds functionality)

### Testing

https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/79bf52af-e1f9-4384-83e4-48e8e37ead9d

I am having issues with clerk so I tested using the main code. Here is a demo until I figure out clerk.

### CODE REVIEW

**Comment by helenaglow** [comment]
I still have to fix lint unfortunately

File: apps/nextjs/package.json [hunk 1]
```diff
@@ -21,8 +21,10 @@
     "@trpc/next": "^10.43.1",
     "@trpc/react-query": "^10.43.1",
     "@trpc/server": "^10.43.1",
+    "@types/react-datepicker": "^4.19.3",
```
Comment (BakerWJ) [inline apps/nextjs/package.json:24]: move to devDependency

File: apps/nextjs/src/components/absence.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface AbsenceProps {
+  data: {
+    userId: number;
+    absenceDate: Date;
+  };
```
Comment (BakerWJ) [inline apps/nextjs/src/components/absence.tsx:7]: probably better to directly expose these fields instead of just putting them in `data`

**Review by BakerWJ — COMMENTED** [review]
looks good overall :)

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**